Mathlib/CategoryTheory/Bicategory/Functor/LocallyDiscrete.lean

```diff
@@ -0,0 +1,89 @@
+/-
+Copyright (c) 2024 Joël Riou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joël Riou
+-/
+import Mathlib.CategoryTheory.Bicategory.Functor.Pseudofunctor
+import Mathlib.CategoryTheory.Bicategory.LocallyDiscrete
+import Mathlib.CategoryTheory.Category.Cat
+
+/-!
+# Constructor for pseudofunctors from a strict locally discrete bicategory
+
+In this file, we define a constructor
+`pseudofunctorOfIsLocallyDiscrete` for the type `Pseudofunctor B C`
+when `C` is any bicategory, and `B` is a strict locally discrete category.
+Indeed, in this situation, we do not need to care about the field `map₂`
+of pseudofunctors because all the `2`-morphisms in `B` are identities.
+
+We also define a specialized constructor `LocallyDiscrete.mkPseudofunctor` when
+the source bicategory is of the form `B := LocallyDiscrete B₀` for a category `B₀`.
+
+-/
+
+namespace CategoryTheory
+
+open Bicategory
+
+/-- Constructor for pseudofunctors from a strict locally discrete bicategory. In that
+case, we do not need to provide the `map₂` field of pseudofunctors. -/
+@[simps obj map mapId mapComp]
+def pseudofunctorOfIsLocallyDiscrete
+    {B C : Type*} [Bicategory B] [IsLocallyDiscrete B] [Bicategory C] [Strict B]
```

Comment:
Do we need `Strict B` here? Should this not already follow from `IsLocallyDiscrete B`?

Reply:
Thanks! Indeed, `IsLocallyDiscrete` implies `Strict`!